website/content/specification/syntax/constraints.md

```diff
@@ -163,6 +163,19 @@ And the following document.
 
 The expect constraint would pass for each `sibling` in the `parent` named "p1", and would fail for each `sibling` in the `parent` named "p2".
 
+## `expect` constraints
+
+The `expect` constraint is a type of Metaschema constraint that restricts field or flag value(s) based on the evaluation of a test Metapath expression.
+
+The `@target` attribute of an `<expect>` constraint specifies the node(s) in a document instance whose value is restricted by the constraint.
+
+The `@test` attribute of an `<expect>` constraint specifies the logical condition to be evaluated against each value node resulting from evaluating the `@target`. This expression must evaluate to [a Metaschema boolean value](/specification/datatypes#boolean) `true` or `false`.
+
+When the `@test` expression evaluates to `true` for a value node, then the value node is considered valid and passes the constraint.
+
+When the `@test` expression evaluates to `false` for a value node, then the value node is considered not valid and fails the constraint. A constraint may have an optional [`@level`](#level) attribute and/or an optional child [`<message>`](#message) element to indicate severity and documentation explaining how the target nodes are invalid.
+
+If defined, the `<message>` value MUST be a [Metaschema string value](/specification/datatypes#string). It may contain a Metapath expression template that starts with `{` and ends with `}`. The expression in the template MUST be set to the result of evaluating the `@expression` using the current node as the Metapath [evaluation focus](#constraint-processing).
```

Comment:
```suggestion
If defined, the `<message>` value MUST be a [Metaschema string value](/specification/datatypes#string). It may contain a Metapath expression templates that start with `{`, contain a Metapath expression, and end with `}`.  When evaluating a template Metapath expression, the context of the Metapath [evaluation focus](#constraint-processing) will be the failing value node.
```

Reply:
Sorry I don't know how I missed this one, fixed in https://github.com/usnistgov/metaschema/pull/493/commits/dd58e570fde8d14996e456c7584b3486697a84a7 and re-resolving.